Docker/python-nginx/python2.7-alpine3.7/nginx.conf

```diff
@@ -0,0 +1,83 @@
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+
+events {
+	worker_connections 100000;
+	# multi_accept on;
+}
+
+http {
+
+	##
+	# Basic Settings
+	##
+
+	sendfile on;
+	tcp_nopush on;
+	tcp_nodelay on;
+	keepalive_timeout 65;
+	types_hash_max_size 2048;
+	# server_tokens off;
+
+	# server_names_hash_bucket_size 64;
+	# server_name_in_redirect off;
+
+	include /etc/nginx/mime.types;
+	default_type application/octet-stream;
+
+	##
+	# SSL Settings
+	##
+	ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
+	ssl_prefer_server_ciphers on;
+
+	##
+	# Logging Settings
+	##
+  log_format json '{"gen3log": "nginx", '
+          '"date_access": "$time_iso8601", '
+          '"user_id": "$http_x_userid", '
+          '"request_id": "$http_x_reqid", '
+          '"session_id": "$http_x_sessionid", '
+          '"visitor_id": "$http_x_visitorid", '
+          '"network_client_ip": "$http_x_forwarded_for", '
+          '"network_bytes_write": $body_bytes_sent, '
+          '"http_response_time": "$request_time", '
+          '"http_status_code": $status, '
+          '"http_request": "$request_uri", '
+          '"http_verb": "$request_method", '
+          '"http_referer": "$http_referer", '
+          '"http_useragent": "$http_user_agent", '
+          '"message": "$request"}';
+
+  log_format aws   '$http_x_forwarded_for - $http_x_userid [$time_local] '
+              '"$request" $status $body_bytes_sent '
+              '"$http_referer" "$http_user_agent"';
+	
+  access_log /var/log/nginx/access.log json;
+  access_log /dev/stdout  json;
```

Comment:
One of these should go - the `Dockerfile` links `/var/log/nginx/access.log` to `stdout` too I think ...